test/testdata/Dockerfile

```diff
@@ -1,9 +0,0 @@
-FROM registry.centos.org/centos/centos:latest
```

Comment:
ok, my previous comment was a bit unfair as this is also an external dependency :) This could be squashed in the commit moving to use quay.io/../nginx though.

Reply:
Right.